tests/queries/skip_list.json

```diff
@@ -0,0 +1,130 @@
+{
```

Comment:
It's is a pity that JSON does not support comments.
(but it's Ok, I cannot propose something better).

Reply:
Maybe Python can parse JSON extension with comments?

Reply:
No vanilla python json module doesn't support comments in JSON :( Also it's possible to move this lisk into `clickhouse-test` script or separate `.py` file and receive config with `eval` or `exec`.

Reply:
Looks like the recommended way to support comments in JSON with Python is JSMin:
https://pypi.org/project/jsmin/